src/screens/Profile/Profile.jsx

```diff
@@ -0,0 +1,248 @@
+    setShowPasswordModal(true);
+  }, []);
+
+  const closePasswordModal = useCallback(() => {
+    setShowPasswordModal(false);
+  }, []);
+
+  const openEmailModal = useCallback(() => {
+    setShowEmailModal(true);
+  }, []);
+
+  const closeEmailModal = useCallback(() => {
+    setShowEmailModal(false);
+  }, []);
+
+  const openDeleteAccountModal = useCallback(() => {
+    setShowDeleteAccountModal(true);
+  }, []);
+
+  const closeDeleteAccountModal = useCallback(() => {
+    setShowDeleteAccountModal(false);
+  }, []);
+
+  useEffect(() => {
+    getProfileStats();
+  });
+
+  //make api call to login
+  const getProfileStats = () => {
+    getStats()
+      .then((response) => {
+        //store stats in variables
+        setStats(response.data);
+      })
+      .catch((event) => {
+        if (event.response?.status === 401 || event.response?.status === 404) {
+          showSnack("error", "Error fetching stats");
+          return;
+        }
+
+        showSnack("error", "Internal Server Error");
+      });
+  };
+
+  const checkDeleteConfirmation = (value) => {
+    if (value === "delete") {
+      setCanSubmitDelete(true);
+    } else {
+      setCanSubmitDelete(false);
+    }
+  };
+
+  const handleLogout = useCallback(() => {
+    dispatch(signOut());
+  }, [dispatch]);
+
+  const columns = React.useMemo(
+    () => [
+      {
+        Header: t("screens.profile.stats.stats"),
+        accessor: "stats", // accessor is the "key" in the data
+      },
+      {
+        Header: t("global.packages"),
+        accessor: "packages",
+      },
+      {
+        Header: t("global.groups"),
+        accessor: "groups",
+      },
+      {
+        Header: t("global.vocabs"),
+        accessor: "vocabs",
+      },
+    ],
+    [t]
+  );
+
+  const data = React.useMemo(
+    () => [
+      {
+        stats: t("screens.profile.stats.total"),
+        packages: stats.languagePackages || "-",
+        groups: stats.inactiveGroups + stats.activeGroups || "-",
+        vocabs: stats.inactiveVocabulary + stats.activeVocabulary || "-",
+      },
+      {
+        stats: t("screens.profile.stats.active"),
+        packages: "-",
+        groups: stats.activeGroups || "-",
+        vocabs: stats.activeVocabulary || "-",
+      },
+      {
+        stats: t("screens.profile.stats.inactive"),
+        packages: "-",
+        groups: stats.inactiveGroups || "-",
+        vocabs: stats.inactiveVocabulary || "-",
+      },
+    ],
+    [
+      stats.activeGroups,
+      stats.activeVocabulary,
+      stats.inactiveGroups,
+      stats.inactiveVocabulary,
+      stats.languagePackages,
+      t,
+    ]
+  );
+
+  return (
+    <>
+      <div className="profile-screen">
+        <div className="profile-avatar-wrapper">
+          <PersonIcon className="profile-avatar" />
+        </div>
+        <h1 className="profile-username">{username}</h1>
+        <Table columns={columns} data={data} pagination={false} />
```

Comment:
maybe we can move this stats table to a single component so that we can easily reuse this if we need.

Reply:
done